Doc/reference/expressions.rst

```diff
@@ -1336,10 +1336,10 @@ built-in types.
 
   The not-a-number values :const:`float('NaN')` and :const:`Decimal('NaN')`
   are special.  They are identical to themselves (``x is x`` is true) but
-  are not equal to themselves (``x == x`` is false).  Additionally,
-  comparing any number to a not-a-number value
+  are not equal to themselves (``x == x`` is false), or any other number.
+  Additionally, all ordered comparison of any number to a not-a-number value
   will return ``False``.  For example, both ``3 < float('NaN')`` and
-  ``float('NaN') < 3`` will return ``False``.
+  ``float('NaN') < 3`` will return ``False``. This is compliant with IEE 754.
```

Comment:
Style nit: Please start a new sentence with two spaces. See line 1338 for example:

```rst
...
are special.  They are identical to themselves (``x is x`` is true) but
...
```